go.mod

```diff
@@ -34,11 +33,7 @@ require (
 	filippo.io/edwards25519 v1.0.0-rc.1 // indirect
 	github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4 // indirect
 	github.com/99designs/keyring v1.2.1 // indirect
-	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
-	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d // indirect
-	github.com/Microsoft/go-winio v0.5.2 // indirect
-	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
-	github.com/Workiva/go-datastructures v1.0.53 // indirect
+	github.com/ChainSafe/go-schnorrkel v1.0.0 // indirect
```

Comment:
This version of `go-schnorrkel` conflicts with the one used by tendermint, which is the cause of most of the errors in the CI. I'd suggest using this at the bottom of go.mod instead:
```
replace github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
```

Then you can use `import "github.com/ChainSafe/go-schnorrkel/1"` anywhere go-schnorrkel is used. Since this is in `go-substrate-rpc-client`, you would need to make the go mod change in that repo (or a fork that is used here) also.

Reply:
The `go-supstrat-rpc-client` is using the [go-subkey](https://github.com/vedhavyas/go-subkey/) as a direct dependency so the `github.com/ChainSafe/go-schnorrkel v1.0.0` is comming as indirect dependency to this repo. If we decide to modify the version, we need to modify the `go-subkey` repo which is using the `go-schnorrkel`.

Reply:
However, I will fork the sub-key and change it in the Composable organization